docs/tutorials/javascript/send-payments/create-offers.md

```diff
@@ -0,0 +1,1091 @@
+
+    standbyBalanceField.value =  (await client.getXrpBalance(standby_wallet.address))
+    operationalBalanceField.value = (await client.getXrpBalance(operational_wallet.address))
+    getOffers()                 
+    client.disconnect()    
+  } // End of oPcreateOffer()
+
+  /***********************************
+  ********** OP Get Offers ***********
+  ***********************************/
+
+async function oPgetOffers() {
+  let results  = "Connecting to the selected ledger.\n"
+  operationalResultField.value = results
+  let net = getNet()
+  results = 'Connecting to ' + net + '....\n'
+  const client = new xrpl.Client(net)
+  await client.connect()
+      
+  results  += "Connected.\n"
+  const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
+  results += operationalNameField.value + " account: " + operational_wallet.address
+  operationalResultField.value = results   
+
+  // -------------------------------------------------------- Prepare request
+
+  results += '\n\n*** Offers ***\n'
+  let offers
+  try {
+    const offers = await client.request({
+      method: "account_offers",
+      account: operational_wallet.address,
+      ledger_index: "validated"
+    })
+    results += JSON.stringify(offers,null,2)
+  } catch (err) {
+      results += err
+  }
+  results += JSON.stringify(offers,null,2)
+  operationalResultField.value = results
+  client.disconnect()
+}// End of oPgetOffers()
+
+/************************************
+ ********** Op Cancel Offer *********
+ ***********************************/
+
+async function oPcancelOffer() {
+    let net = getNet()
+    let results = 'Connecting to ' + net + '....\n'
+    const client = new xrpl.Client(net)
+    await client.connect()
+        
+    results  += "Connected.\n"
+    const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
+    results += "wallet.address: = " + operational_wallet.address
+    operationalResultField.value = results
+  
+  // -------------------------------------------------------- Prepare transaction
+  
+  /* OfferSequence is the Seq value when you getOffers. */
+  const prepared = await client.autofill({
+  "TransactionType": "OfferCancel",
+  "Account": operational_wallet.address,
+  "OfferSequence": parseInt(operationalOfferSequenceField.value)
+})
+  
+// ------------------------------------------------- Sign prepared instructions
+  const signed = operational_wallet.sign(prepared)
+  
+// -------------------------------------------------------- Submit signed blob
+  const tx = await client.submitAndWait(signed.tx_blob)
+      
+  results  += "\nBalance changes: \n" + tx.result + "\n" +
+    JSON.stringify(xrpl.getBalanceChanges(tx.result.meta), null, 2)
+  operationalResultField.value = results
+
+  operationalBalanceField.value = (await client.getXrpBalance(operational_wallet.address))                 
+  client.disconnect()    
+} // End of oPcancelOffer()
+```
+
+## ripplex3b-name-field-support.js
```

Comment:
I'm not sure this is the best place to introduce the name field support. This is the only tutorial that supports this functionality; I'd like to see this introduced in the very first quickstart tutorial and refactored into subsequent ones.

Reply:
I agree. We need to set a priority for the refactor - if it's sooner than later, we should incorporate named fields from Module one. If not, we'll have to go ahead and introduce it here, even if it's not optimal.

Reply:
That's a good point. I'm not sure we'll be able to refactor any time soon, so I'll go ahead and approve.